config/boot.rb

```diff
@@ -1,6 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
-# https://github.com/Shopify/bootsnap/issues/110
-ENV['BOOTSNAP_CACHE_DIR'] = File.expand_path("../tmp/cache#{ENV['TEST_ENV_NUMBER']}", __dir__) if ENV['RAILS_ENV'] == 'test'
```

Comment:
This was required to fix a common flakyness issue in Check API tests... it's not needed anymore because `bootsnap` was upgraded as per change above?

Reply:
yeah, that's my understanding based on what's described in the issue in the comment!